core/org.eclipse.birt.core/src/org/eclipse/birt/core/util/EclipseUtil.java

```diff
@@ -0,0 +1,27 @@
+package org.eclipse.birt.core.util;
+
+import org.eclipse.core.runtime.Platform;
+import org.osgi.framework.Bundle;
+
+/**
+ * Eclipse and OSGi utilities
+ *
+ * @since 4.9
+ *
+ */
+public class EclipseUtil {
```

Comment:
`EclipseUtil` is too generic. 

This could be 
```
public final classs FindBundle implements Function<String, Optional<Bundle>> {
...
}
```

Reply:
It is meant to hold more platform access classes in the future. one-class-one-method is overkill. WDYT?

Reply:
I think we need to use platform facilities directly and improve the platform capabilities rather than invent a (potentially big) utilities class with static method. Of cause this is the question of style and the style of PR is closer to current BIRT source base. Let's go forward.